zipline/testing/core.py

```diff
@@ -853,22 +539,32 @@ def write_minute_data_for_asset(env, writer, start_dt, end_dt, sid,
         for minute in minute_blacklist:
             df.loc[minute] = 0
 
-    writer.write(sid, df)
+    return df
+
+
+def write_minute_data_for_asset(env, writer, start_dt, end_dt, sid,
```

Comment:
This seems like it's just shuffling arguments around.  Is this just for backward compat with something else?